package.json

```diff
@@ -19,7 +19,9 @@
     "hoek": "3.x.x",
     "joi": "7.x.x",
     "oppsy": "1.x.x",
-    "pumpify": "1.3.x"
+    "pumpify": "1.3.x",
+    "traverse": "0.6.6",
+    "wreck": "7.x.x"
```

Comment:
Should Wreck be pinned to `7.2.x`? since that is the version were the Wreck EventEmitter was added to the process? See hapijs/wreck#125